incluster_eventing/emitter/config.yaml

```
@@ -0,0 +1,14 @@
+name: event-emitter
+namespace: default
+runtime: nodejs14
+source:
+    sourceType: inline
+apiRules:
+    - name: incomming-http-trigger
+      service:
+        host: incomming
```

```suggestion
    - name: incoming-http-trigger
      service:
        host: incoming
```
Noticed a typo here.